tweet_annual_days.py

```diff
@@ -7,6 +7,7 @@
 from datetime import date
 from tweet import authenticate_api
 from words_dir.tweet_annual_events import events
+from tweet_history import tweet_historicat_event
```

Comment:
This import actually doesn't belong. The function call was an oversight on my part. I meant that to be `tweet_annual_event()`.